src/common/xo-week-charts/index.js

```diff
@@ -246,21 +246,18 @@ class XoWeekChart extends Component {
       .select(this.refs.chart)
       .append('svg')
       .attr('transform', `translate(${HORIZON_AREA_MARGIN}, 0)`))
-
     ;svg
```

Comment:
Can you remove all these `;` and reformat please?

Reply:
Prettier doesn't let me remove them, probably because of the `::` syntax.